contracts/RedstoneProxy.sol

```diff
@@ -46,4 +49,19 @@ contract RedstoneProxy is RedstoneCoreProxy, ERC1967Upgrade {
     function _priceFeed() internal view override returns (address) {
         return StorageSlot.getAddressSlot(_PRICE_FEED_SLOT).value;
     }
+
+    
+    function _setPriceFeed(address _priceFeedAddress) private {
+        require(_priceFeedAddress != address(0), "Price feed address cannot be empty");
+        
+        assert(_IMPLEMENTATION_SLOT == bytes32(uint256(keccak256("eip1967.proxy.implementation")) - 1));
+        StorageSlot.getAddressSlot(_PRICE_FEED_SLOT).value = _priceFeedAddress;
+    }
+
+    /**
+     * @dev Sets the price feed address (internal function)
+     */
+    function setPriceFeed(address _priceFeedAddress) external onlyOwner {
```

Comment:
Shouldn't that be managed by admin account rather by owner (and without introducing Ownable), in a pattern similar to :
https://github.com/redstone-finance/redstone-flash-storage/blob/master/contracts/RedstoneUpgradeableProxy.sol#L97